WordPress/src/test/java/org/wordpress/android/ui/mysite/quickstart/QuickStartBlockBuilderTest.kt

```diff
@@ -147,6 +128,31 @@ class QuickStartBlockBuilderTest : BaseUnitTest() {
                 )
     }
 
+    /* TASK TYPE ITEM PROGRESS BAR */
+
+    @Test
+    fun `given non zero completed tasks, when block is built, then completed tasks progress is non zero`() {
+        val quickStartBlock = buildQuickStartBlock()
+
+        assertThat(getQuickStartTaskTypeItem(quickStartBlock).progress)
+                .isEqualTo(((completedTasks.size / (completedTasks.size + uncompletedTasks.size.toFloat())) * 100)
```

Comment:
Minor (🔍): I would avoid using the same formula to verify correctness, instead what I believe is better is to simply use a constant, in this case the number `50`. Copy pasting the formula within test to verify main code's correctness is something I was doing as well, but I keep finding that this is not a good way of verifying correctness since it might produce false positives. It is a bit hard for me to explain exactly why that is, and doing so in a couple of sentences, but the bottom line is that I would personally just use the raw result number to verify this logic. But, it is totally up to you and you might actually had a very valid reason for doing so.

Reply:
In 4895b58